debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/J9ObjectFieldOffsetIterator_V1.java

```diff
@@ -212,7 +212,7 @@ private void fieldOffsetsFindNext() throws CorruptDataException {
 					if (modifiers.anyBitsIn(J9FieldFlagObject)) {
 						if (valueTypeHelper.isFlattenableFieldSignature(J9ROMFieldShapeHelper.getSignature(localField))) {
 							J9ClassPointer fieldClass = valueTypeHelper.findJ9ClassInFlattenedClassCacheWithFieldName(instanceClass, J9ROMFieldShapeHelper.getName(localField));
-							if (valueTypeHelper.isJ9ClassIsFlattened(fieldClass)) {
+							if (valueTypeHelper.isJ9FieldIsFlattened(fieldClass, localField)) {
```

Comment:
Consider updating these terribly-named functions - the DDR names are much better.

Reply:
Ah, these are all DDR - what's the difference between this and `isFieldInClassFlattened`?

Reply:
`isFieldInClassFlattened()` is more like a wrapper to `isJ9FieldIsFlattened()`. `isFieldInClassFlattened()` makes sure flattenedClassCache is not null and the class is in the flattenedClassCache before calling into `isJ9FieldIsFlattened()`. 

I can rename them to `isFieldInClassFlattenedWrapper()` and `isFieldInClassFlattened()` if you want.

Reply:
Let's leave it for now.